src/db/match.js

```diff
@@ -42,6 +42,14 @@ const Match = class {
     return this._update('goals_b', goals)
   }
 
+  get mode () {
+    return this._select('mode')
```

Comment:
I think (might be wrong):
- mode 1v1/2v2 can be inferred  from the  User_in_Match table (see this.usersOfTeam)
- The mode needs to be saved in tournament table (for some tournament modes), it is necessary to create the next round while the turnament is running